frontend/public/components/monitoring/query-browser.tsx

```diff
@@ -120,6 +120,7 @@ const SpanControls: React.FC<SpanControlsProps> = React.memo(
           value={text}
         />
         <Dropdown
+          ariaLabel="graph timespan dropdown"
```

Comment:
Curious if "dropdown" is preferred to "menu" or "dropdown menu".  Not sure what is most helpful to someone using a screen reader.  

It might be interesting to document a standard set of labels to use consistently for `ariaLabel`, etc.

Reply:
Great question, finding the best label can often be a pretty subjective decision so I double checked with some others knowledgable with accessibility to hash out some thoughts on that. I’m actually considering taking off the word dropdown and just making it `graph timespan`. The thought behind that is that roles, properties, and states of an element should probably be communicated through other aria attributes and roles, rather than in the label. I originally had concerns about making both the input and the button have an aria label of “graph timespan”, but in VO it reads as "30m graph timespan edit text" for the input field and "graph timespan collapsed pop up button" for the button. I’m thinking that might be enough context to show what these elements do. What do you think?

Also, I completely agree that we should document the labels for consistency. I know that we had the same concern with PatternFly so that’s a good point. :) What we ended up deciding with PF was if you're attaching that aria-* attribute to the root element of the component, just use the normal jsx/html version that's dash delimited, if you're sending that value to a child component of the component, call it out like submiBtnAriaLabel or graphTimelineDropdownAriaLabel. I’m super open to how you guys prefer these labels to be used here!

Reply:
@jessiehuff `graph timespan` sounds good to me. if you push an update, we can tag.